tests/ethereum/main.rs

```diff
@@ -93,12 +95,34 @@ fn build_tx_env(
         gas_priority_fee: tx.max_priority_fee_per_gas,
         blob_hashes: tx.blob_versioned_hashes.clone(),
         max_fee_per_blob_gas: tx.max_fee_per_blob_gas,
-        authorization_list: None, // TODO: Support in the upcoming hardfork
+        //authorization_list: None,
+        authorization_list: Some(AuthorizationList::Signed(
+            tx.authorization_list
+                .iter()
+                .map(|auth| {
```

Comment:
Do we have a reference for this implementation, maybe in `revm`?

Reply:
My implementation was from my own judgement.
However, I could find similar implementation in `bluealloy/revm` repo, that looks standard. https://github.com/bluealloy/revm/blob/dad766d4949191a7368360e1b7dc1f3a45040b2c/bins/revme/src/cmd/statetest/runner.rs#L401
It uses `TestAuthorization` struct which implements `into_recovered` function.
In `risechain/revm` repo, we use `Authorization` instead of `TestAuthorization` and we don't have `into_recovered` function implemented.
I think we can implement the function for `Authorization` struct in our `risechain/revm` repo, and use their implementation.
Please let me know your thoughts.

Reply:
Yeah, we can cherry-pick https://github.com/bluealloy/revm/commit/be1d324298b6a1e20f8b17aff34f95206304117b in `risechain/revm` but ideally we'll get it in the next rebase on a new `revm` release. In the meantime, we can replicate the `into_recovered` here? It's pretty short:
https://github.com/bluealloy/revm/blob/dad766d4949191a7368360e1b7dc1f3a45040b2c/crates/statetest-types/src/test_authorization.rs#L29-L43